src/WingetCreateCLI/Commands/UpdateLocaleCommand.cs

```diff
@@ -0,0 +1,291 @@
+        /// Executes the update-locale command flow.
+        /// </summary>
+        /// <returns>Boolean representing success or fail of the command.</returns>
+        public override async Task<bool> Execute()
+        {
+            CommandExecutedEvent commandEvent = new CommandExecutedEvent
+            {
+                Command = nameof(UpdateLocaleCommand),
+                Id = this.Id,
+                Version = this.Version,
+                HasGitHubToken = !string.IsNullOrEmpty(this.GitHubToken),
+            };
+
+            Prompt.Symbols.Done = new Symbol(string.Empty, string.Empty);
+            Prompt.Symbols.Prompt = new Symbol(string.Empty, string.Empty);
+
+            string exactId;
+            try
+            {
+                exactId = await this.GitHubClient.FindPackageId(this.Id);
+            }
+            catch (Octokit.RateLimitExceededException)
+            {
+                Logger.ErrorLocalized(nameof(Resources.RateLimitExceeded_Message));
+                return false;
+            }
+
+            if (!string.IsNullOrEmpty(exactId))
+            {
+                this.Id = exactId;
+            }
+
+            List<string> manifestContent;
+
+            try
+            {
+                manifestContent = await this.GitHubClient.GetManifestContentAsync(this.Id, this.Version);
+                Manifests originalManifests = Serialization.DeserializeManifestContents(manifestContent);
+
+                // Validate input locale argument and switch command flow if applicable.
+                if (!string.IsNullOrEmpty(this.Locale))
+                {
+                    try
+                    {
+                        if (LocaleHelper.GetMatchingLocaleManifest(this.Locale, originalManifests) == null)
+                        {
+                            Logger.ErrorLocalized(nameof(Resources.LocaleDoesNotExist_Message), this.Locale);
+
+                            // Switch to new locale flow if user accepts.
+                            if (Prompt.Confirm(Resources.SwitchToNewLocaleFlow_Message))
+                            {
+                                NewLocaleCommand command = new NewLocaleCommand
+                                {
+                                    Id = this.Id,
+                                    Version = this.Version,
+                                    Locale = this.Locale,
+                                    GitHubToken = this.GitHubToken,
+                                };
+                                await command.LoadGitHubClient();
+                                Console.WriteLine();
+                                return await command.Execute();
+                            }
+                        }
+                    }
+                    catch (ArgumentException)
+                    {
+                        Logger.ErrorLocalized(nameof(Resources.InvalidLocale_ErrorMessage));
+                        return false;
+                    }
```

Comment:
Remove, we don't support switching flows from update -> new

Reply:
Removed support for changing command flows in the latest commit. Added error message to say to user "Use the update-locale command to update an existing locale manifest"/ "Use the new-locale command to generate a new locale manifest"